stdlib/src/collections/string.mojo

```diff
@@ -1038,37 +1039,6 @@ struct String(
         write_args(output, args, sep=sep, end=end)
         return output^
 
-    @staticmethod
-    @always_inline
-    fn _from_bytes(owned buff: UnsafePointer[UInt8]) -> String:
-        """Construct a string from a sequence of bytes.
-
-        This does no validation that the given bytes are valid in any specific
-        String encoding.
-
-        Args:
-            buff: The buffer. This should have an existing terminator.
-        """
-
-        return String(ptr=buff, length=len(StringRef(ptr=buff)) + 1)
```

Comment:
**Suggestion** Can you just use `StringSlice` here to replace the `StringRef`?  We don't want to delete these `_from_bytes` methods quite yet as we use them internally in some MAX code.

Reply:
I just added them again, since we'd need to reimplement `strlen` if we wanted to avoid using `StringRef` here.

Reply:
👍 thanks!